client/src/types/CommentData.ts

```diff
@@ -0,0 +1,7 @@
+export type CommentData = {
+  id: number;
+  solutionId: number;
+  userId: number;
+  text: string;
+  helpful: boolean;
```

Comment:
What is `helpful`? If it's like users assessing whether a comment is good or not by boolean I don't think it's the way to go. I think it will be better to have a score system, like Piazza where users can click on "good comment" and the score updates

Reply:
I agree, similar to reddit upvotes